### PR TITLE
fix sanity check for xtrans

### PR DESCRIPTION
**Pull request by boegel**
list of files in `sanity_check_paths` was incorrectly expanded in #2504 

reported by @hajgato

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/81a84f4621c82aed52a2 for a full test report.

File: easybuild/easyconfigs/x/xtrans/xtrans-1.3.5-goolf-1.5.14.eb [hunk 1]
```diff
@@ -15,9 +15,10 @@ sources = [SOURCE_TAR_GZ]
 source_urls = [XORG_LIB_SOURCE]
 
 sanity_check_paths = {
-    'files': ['include/X11/Xtrans/%s' % x for x in ['transport.c', 'Xtrans.c', 'Xtransdnet.c', 'Xtrans.h',
-                                                    'Xtransint.h', 'Xtranslcl.c', 'Xtransos2.c', 'Xtranssock.c',
-                                                    'Xtranstli.c', 'Xtransutil.c']],
+    'files': ['include/X11/Xtrans/%s' % x for x in ['transport.c', 'Xtrans.c', 'Xtrans.h', 'Xtransint.h', 'Xtranslcl.c',
+                                                    'Xtranssock.c', 'Xtransutil.c']],
+    'files': ['include/X11/Xtrans/%s' % x for x in ['transport.c', 'Xtrans.c', 'Xtrans.h', 'Xtransint.h',
```
Comment (hajgato) [inline easybuild/easyconfigs/x/xtrans/xtrans-1.3.5-goolf-1.5.14.eb]: @beoegl: double `'files':`

**Comment by hajgato** [comment]
@boegel: at my side, good to go!

**Comment by boegel** [comment]
Going in, thanks for the review @hajgato!

**Pull request merged**